node/src/BaseClient.ts

```diff
@@ -297,6 +297,35 @@ export type DecoderOption = {
     decoder?: Decoder;
 };
 
+/** A replacement for `Record<GlideString, T>` - array of key-value pairs. */
+export type GlideRecord<T> = { key: GlideString; value: T }[];
+
+// GLideRecord for score for sorted set commands?
+// TODO export
+export type ElementScoreData = { element: GlideString; score: number }[];
```

Comment:
Suggest to call it SortedSetDataType or something similar.  
For Stream, we can call it StreamEntryDataType?